src/hooks/useUser.js

```diff
@@ -12,7 +11,7 @@ const useUser = () => {
     const formData = new FormData();
     formData.append("email", email);
     formData.append("password", password);
-    const response = await axios.post(`/api/${currentConfig.login}`, formData);
+    const response = await axiosPrivate.post(`/api/${currentConfig.login}`, formData);
```

Comment:
Here we should use axios only as the axiosPrivate one adds listeners to the request and also adds an authorised header too. And that all is not needed here

Reply:
"test": {
    "globalSetting": "global_setting",
    "contentData": "contents",
    "pageConfig": "page_configs",
    "login": "https://holydemo.com/api/login"
  }, We have to set  login  , full url path , because if we set path only  , axios redirects to 127.0.0.1,  if you have other options , it is welcome

Reply:
Why do we need it to be in the config ?

Reply:
If we are using seperate login function then just use it hardcoded there

Reply:
I get your concern but this is just for test env. either way it's okey! nothing serious !